### PR TITLE
Show Members under each Minister

### PR DESCRIPTION
**Pull request by mekbib-tekle**

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -11,3 +11,70 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```
Comment (ermyas7) [inline frontend/src/index.css:14]: Could we use a styled component here?

File: frontend/src/utils/api.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    });
+    const data = await response.json();
+
+    return data;
+};
+
+export const handleLogout = async () => {
+    try {
+      // TODO change this endpoint to /logout, handle response
+      await fetch('http://localhost:8000/auth/profile', {
```
Comment (ermyas7) [inline frontend/src/utils/api.ts:17]: We could replace the url with BASE_URL here too.

File: frontend/src/components/FollowUp.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import { useEffect, useState } from 'react';
+import { MinisterResponse, Member, MemberObj, Minister } from '../types/Member';
+import { getWeekMap } from '../utils/date';
+import { get } from '../utils/api';
+import { Grid, Typography } from '@mui/material';
+import MemberGroupsByContactLog from './MemberGroupsByContactLog';
+
+// convert the data from the server to a more structured format
+const mapResponse = (data: MinisterResponse[]): Minister[] => {
+    const minsiterData = data && data.map((minister: MinisterResponse): Minister => {
```
[the remark this answers is not in the document]
Comment (mekbib-tekle) [reply frontend/src/components/FollowUp.tsx:11]: minister object is actually needed in the function.

File: frontend/src/index.css (hunk 1 above)
Comment (mekbib-tekle) [reply frontend/src/index.css:14]: i don't know if i am being old fashioned, but i like the good old className and separate css more than styled components (which i haven't been able to get used to). what do you say?

**Pull request merged**